CONTRIBUTING.md

```diff
@@ -2,57 +2,95 @@
-* New features must have user documentation.
-* All new code, both bugfixes and new features, must have tests.
-* Pull requests must be cleanly rebased on top of master without multiple branches mixed together.
-
-Please read the [review checklist](https://osbs.readthedocs.io/en/latest/contributors.html#submitting-changes).
+- We all know that there is an exception to every rule. Sometimes it might
+  happen that your pull request doesn't satisfy all the below points. This is
+  fine, assuming there's a good reason for this ― and assuming you tell us the
+  reason.
+- If you want to implement major new functionality, we'd like to ask you to open
+  a bug with proposal to discuss it with us. We're nice folks and we don't bite
+  ― we just want to see what you're up to and have some suggestions to save your
+  time as well as ours.
+- New features must have user documentation.
+- All new code, both bugfixes and new features, must have tests.
+- Pull requests must be cleanly rebased on top of master without multiple
+  branches mixed together.
+
+Please read the [review checklist][].
 
 ## Code Quality
 
 We try to write good code. This is what it means to us:
 
-* Maximum line length is 100 characters
-* Pull Request *must* pass following checks:
-  * [Travis CI test](https://travis-ci.org/containerbuildsystem/atomic-reactor) - This means that no tests fail on Python 2.7 and >= 3.4
-    * You can run tests locally using `pytest`
-  * [Landscape test](https://landscape.io/github/containerbuildsystem/atomic-reactor) - This means that code follows pep8 and is otherwise sane. To check this, run this command:
-    * `flake8 atomic_reactor`
-    * Landscape runs some more tests, which aren't reproducible locally (as far as we know). Feel free to rebase your pull request if Landscape finds issues that you couldn't find while testing on your machine
-* Pull Request *should* pass following checks:
-  * [Coveralls test](https://coveralls.io/r/containerbuildsystem/atomic-reactor) - This means that code coverage doesn't decrease. You can check that by running `py.test --cov atomic_reactor` without your patch and then with it. Coveralls tend to give false positives, so even if they report failure, we will still accept your pull request assuming it has decent tests
-* Methods and functions that are to be called from different modules in Atomic Reactor must have docstrings.
-* Code should be readable, meaning:
-  * Comment where appropriate
-  * Separate logical parts of methods/functions by newline
-  * Use meaningful names for methods/functions/classes/variables
+- Maximum line length is 100 characters
+- Pull Request *must* pass following checks
+  - [Travis CI test][]
+    - This means that no tests fail on Python 2.7 and >= 3.4
+    - You can run tests locally using `pytest`
+  - [Landscape test][]
```

Comment:
This is dead, it may be removed in another PR

Reply:
Yeah, I left the *content* completely untouched (and if I didn't, it's a bug :) )...